docs/advanced-topics/docker-alternatives.rst

```diff
@@ -73,6 +73,36 @@ add a line to your ``~/.dockstore/config``:
 
     cromwell-vm-options: -Dconfig.file=<absolute path to your Cromwell conf>
 
+Podman
+---------------
+
+Podman is a daemon-less alternative of Docker that allows end users to run docker/OCI containers without root privileges.
```

Comment:
We should probably define with daemon means in this context in the Dockstore dictionary.

Reply:
I can create a PR to add this to the dictionary. How is this definition of daemon? "Daemon is a background process that is used to create images and run containers. The Docker daemon constantly listens for Docker API requests and processes them."